helper/impls/helper-impl-110/src/main/java/com/linkedin/avroutil1/compatibility/avro110/Avro110Adapter.java

```diff
@@ -88,6 +92,10 @@ private void tryInitializeCompilerFields() {
             Class<?> fieldTypeEnum = Class.forName("org.apache.avro.generic.GenericData$StringType");
             Field charSequenceField = fieldTypeEnum.getDeclaredField("CharSequence");
             charSequenceStringTypeEnumInstance = charSequenceField.get(null);
+            Field javaLangStringField = fieldTypeEnum.getDeclaredField("String");
+            javaLangStringTypeEnumInstance = javaLangStringField.get(null);
+            Field utf8Field = fieldTypeEnum.getDeclaredField("Utf8");
+            utf8TypeEnumInstance = utf8Field.get(null);
```

Comment:
IMHO, these should be params of your `StringRepresentation` enum

Reply:
they (also) are, but these are instances of an avro  enum needed for reflection later